stored/src/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,92 @@
+import prisma from "@/lib/prisma";
+import { PrismaAdapter } from "@next-auth/prisma-adapter";
+import { AuthOptions } from "next-auth";
+import NextAuth from "next-auth/next";
+import GithubProvider from "next-auth/providers/github";
+import CredentialsProvider from "next-auth/providers/credentials";
+import bcrypt from "bcrypt";
+
+if (!process.env.GITHUB_ID || !process.env.GITHUB_SECRET) {
```

Comment:
Did you create a 0Auth account in Github or got some valid Github Id and Secret?